x-pack/plugin/rollup/src/main/java/org/elasticsearch/xpack/rollup/action/TransportPutRollupJobAction.java

```diff
@@ -102,6 +111,25 @@ protected void masterOperation(
         XPackPlugin.checkReadyForXPackCustomMetadata(clusterState);
         checkForDeprecatedTZ(request);
 
+        int numberOfCurrentRollupJobs = RollupUsageTransportAction.findNumberOfRollupJobs(clusterState);
+        if (numberOfCurrentRollupJobs == 0) {
+            try {
+                boolean hasRollupIndices = hasRollupIndices(clusterState);
+                if (hasRollupIndices == false) {
+                    listener.onFailure(
+                        new IllegalArgumentException(
+                            "rollup has been deprecated and will be removed, therefor starting "
+                                + "from 8.15.0, creating new rollup jobs is no longer allowed in clusters that don't have any rollup usage."
```

Comment:
therefore*

I'd just say: "new rollup jobs are not allowed in clusters that don't have any rollup usage, since rollup has been deprecated". The details should be included in rollup documentation.